### DISCUSSION TITLE
Add ability to set anchor to other skin elements

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#26020] [Ideas]
Would be nice to add the ability to anchor skin elements... to other skin elements

I want to add a combo counter + max combo counter to the bottom left and alight it to the edge of the progress bar, but this requires me to go in and out of the skin editor as when I go into normal gameplay, the counters move

Here's a clip of what I'm frustrated with
https://github.com/ppy/osu/assets/66184829/0346377d-53fb-4a52-b397-1d896f80b811

I would love if I could set the anchor of those counters to the top left corner of the progress bar, so that it's easier to align stuff

**Comment by bytenekos** [comment]
To add to this, maybe an easy way to toggle the playing field so that it goes full screen, allowing easier adjustment of skin elements so that "what you see is what you get"

**Reply by Projosh00** [reply]
I think this is something we very much need, without it some elements might overlap in some scenarios.

This is a layout I made for the bottom center of my hud.
![image](https://github.com/ppy/osu/assets/135391814/af10bfd6-d3ec-4427-8287-c6bb322a9937)

It works and looks perfectly fine on 4 keys, but as soon as you switch to 6k or 7k, or any other amount of keys..
![image](https://github.com/ppy/osu/assets/135391814/ddd1fd8e-bf51-4f8f-9040-724f1f6b40bb)
